src/pages/StatisticsChart/block/AverageBlockTime.tsx

```diff
@@ -68,8 +68,7 @@ const useOption = (
             )}</div>`
             dataList.forEach(data => {
               assertSerialsItem(data)
-              assertSerialsDataIsString(data)
-              result += parseTooltip(data)
+              result += parseTooltip({ ...data })
```

Comment:
The types here, including a few other similar places, were wrong before, leading to an incorrect implementation of `assertSerialsDataIsString` during the previous refactoring. It needs to be changed to `assertSerialsDataIsStringArray`.

This assert cannot be directly removed, as it would implicitly pass `any` to `parseTooltip`.

Reply:
The problem is `exceptions are not properly handled`. The final solution should be
1. add proper assertion
2. catch exception and send it to sentry

These refactorings could be done later to fix the online issue fast.

Reply:
That makes sense. Since this is an emergency fix, we can deploy it first and then deal with it.